docs/pages/machine-workload-identity/mwi-use-cases/mwi-use-cases.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Machine & Workload Identity Use Cases
+description: Common use cases with Machine ID & Workload Identity
+labels:
+ - conceptual
+ - mwi
+---
+
+Machine & Workload Identity can work together to create a comprehensive security model. Machines can securely access resources while workloads communicate securely with each other and external services, 
+all managed through Teleport's unified access plane.
```

Comment:
Isn't the _access plane_ old marketing nomenclature that we're no longer using? I thought we were the infrastructure identity platform now?